libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackAdapter.cs

```diff
@@ -19,17 +19,22 @@ namespace Microsoft.Bot.Builder.Adapters.Slack
 {
     public class SlackAdapter : BotAdapter, IBotFrameworkHttpAdapter
     {
+        private const string PostMessageUrl = "https://slack.com/api/chat.postMessage";
+        private const string PostEphemeralMessageUrl = "https://slack.com/api/chat.postEphemeral";
+        private const string SlackOAuthUrl = "https://slack.com/oauth/authorize?client_id=";
+
         private readonly SlackAdapterOptions _options;
         private readonly SlackClientWrapper _slackClient;
-        private readonly string _slackOAuthUrl = "https://slack.com/oauth/authorize?client_id=";
+
         private string _identity;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="SlackAdapter"/> class.
         /// Create a Slack adapter.
         /// </summary>
+        /// <param name="slackClient">An initialized instance of the SlackClientWrapper class.</param>
         /// <param name="options">An object containing API credentials, a webhook verification token and other options.</param>
-        public SlackAdapter(SlackAdapterOptions options)
+        public SlackAdapter(SlackClientWrapper slackClient, SlackAdapterOptions options)
```

Comment:
The SlackAdapterOptions should be part of the SlackClientWrapper, not the adapter, but, let's merge this and keep that refactor for later.

Reply:
I knew that. But since there's an upcoming task for moving these methods around I'd leave it for then